velox/functions/prestosql/tests/MapTest.cpp

```diff
@@ -97,6 +97,17 @@ TEST_F(MapTest, noNulls) {
 
   auto result = evaluate("map(c0, c1)", makeRowVector({keys, values}));
   assertEqualVectors(expectedMap, result);
+
+  // Empty map constructor. We use unknown types here because no way to specify
+  // the output type for the map() call.
+  const TypePtr& emptyMapType =
+      MAP(CppToType<UnknownValue>::create(), CppToType<UnknownValue>::create());
+  SelectivityVector rows(size);
+  auto emptyMapVector =
+      std::make_shared<MapVector>(pool_.get(), emptyMapType, rows);
```

Comment:
Does this create a valid vector? Won't offsets and sizes be null?

Reply:
@mbasmanova It does create a valid vector (from what I can understand). I have added a new constructor for creating "empty" map vectors.